whiskey.yml

```diff
@@ -44,55 +32,44 @@ Build:
     OnlyDuring: Clean
     Path: reset.ps1
 
-- GetPowerShellModule:
-    Name: Carbon.Core
-    Version: 1.0.0
-    Path: Carbon.Cryptography\Modules
-
-- Pester4:
-    Script: Tests\*.Tests.ps1
-
-# Build runs twice on some servers, so delete any ZIP file created by another build.
 - Delete:
-    Path: .output\*.zip
+    OnlyBy: BuildServer
+    Path: Carbon.Cryptography\prism.*
+
+- CopyFile:
+    Path:
+    - CHANGELOG.md
+    - README.md
+    - LICENSE
+    - NOTICE
+    DestinationDirectory: Carbon.Cryptography
 
 - Zip:
-    ArchivePath: .output\Carbon.Cryptography.zip
+    ArchivePath: .output\Carbon.Cryptography-$(WHISKEY_SEMVER2).zip
     Path: 
     - Carbon.Cryptography
+    Exclude:
+    - "*.pdb"
+    - "*.orig"
+    - "prism.*"
 
-Publish:
-
-- AppVeyorWaitForBuildJobs:
-    IfExists: env:PUBLISH
-    UnlessExists: env:APPVEYOR_PULL_REQUEST_NUMBER
-    ApiKeyID: AppVeyorBearerToken
+- Exec:
+    OnlyBy: BuildServer
+    Path: appveyor
+    Argument: [ 'PushArtifact', '.output/Carbon.Cryptography-$(WHISKEY_SEMVER2).zip', '-DeploymentName', 'GitHub' ]
 
 - PublishPowerShellModule:
-    IfExists: env:PUBLISH
-    UnlessExists: env:APPVEYOR_PULL_REQUEST_NUMBER
-    RepositoryName: PSGallery
-    RepositoryUri: https://powershellgallery.com/api/v2/
     Path: Carbon.Cryptography
-    ApiKeyID: PowerShellGalleryApiKey
 
-- SetVariableFromPowerShellDataFile:
-    Path: Carbon.Cryptography\Carbon.Cryptography.psd1
-    Variables:
-        PrivateData:
-            PSData:
-                ReleaseNotes: RELEASE_NOTES
+- Exec:
+    OnlyBy: BuildServer
+    Path: appveyor
+    Argument: [
+        'PushArtifact',
+        '.output/Carbon.Cryptography.$(WHISKEY_SEMVER2_NO_BUILD_METADATA).nupkg',
+        '-DeploymentName',
+        'PowerShellGallery'
+    ]
 
-- GitHubRelease:
-    IfExists: env:PUBLISH
-    UnlessExists: env:APPVEYOR_PULL_REQUEST_NUMBER
-    RepositoryName: webmd-health-services/Carbon.Cryptography
-    ApiKeyID: GitHubAccessToken
-    Tag: $(WHISKEY_SEMVER2_NO_BUILD_METADATA)
-    Commitish: $(WHISKEY_SCM_COMMIT_ID)
-    Name: $(WHISKEY_SEMVER2_NO_BUILD_METADATA)
-    Description: $(RELEASE_NOTES)
-    Assets:
-    - Path: .output\Carbon.Cryptography.zip
-      ContentType: application/zip
-      Name: Carbon.Cryptography-$(WHISKEY_SEMVER2_NO_BUILD_METADATA).zip
\ No newline at end of file
+- Pester4:
+    Script: Tests\*.Tests.ps1

```

Comment:
We're running tests after publishing?

Reply:
For debugging. Since the module files get merged into the .psm1, stack traces become less useful unless you have the ginormous .psm1 file to see where something is failing.

Reply:
Ok then just to clarify: The `appveyor PushArtifact <path> -DeploymentName PowerShellGallery` command does not publish to the PowerShellGallery? It simply stages the artifact somewhere in AppVeyor?